client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -97,10 +104,21 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 	}
 }
 
-export default connect( ( state, { siteId } ) => {
+export default connect( ( state, { siteId, steps } ) => {
 	const tasks = compact( [] );
+	const stepsCompleted = reduce(
+		steps,
+		// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
```

Comment:
We could avoid that by introducing another selector, but is it worth it? 🤔

Reply:
https://www.youtube.com/watch?v=UODX_pYpVxk